db-engine/src/main/java/com/splicemachine/db/impl/jdbc/TransactionResourceImpl.java

```diff
@@ -567,6 +567,9 @@ else if (sqlState.equals(SQLState.LANG_CHECK_CONSTRAINT_VIOLATED)) {
         else if (sqlState.equals(SQLState.LANG_FK_VIOLATION)) {
             se.setSeverity(-530);
         }
+        else if (sqlState.equals(SQLState.LANG_INVALID_FK_COL_FOR_SETNULL)) {
```

Comment:
Is it possible to add IT for this error code? For SQLState.LANG_FK_VIOLATION there are at least some tests in LoadReplaceModeIT.

Reply:
I suggest that we Jira this as a separate item if it's going to take very long at all to construct the IT.

Reply:
Thanks, I added a test for `SQLState.LANG_FK_VIOLATION` in `ForeignKeyActionIT`.